libs/cardano-ledger-test/cardano-ledger-test.cabal

```diff
@@ -61,21 +61,22 @@ library
         array,
         bytestring,
         cardano-crypto-class,
-        cardano-ledger-allegra >=1.1,
-        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.1,
-        cardano-ledger-alonzo-test >=1.1,
-        cardano-ledger-babbage:{cardano-ledger-babbage, testlib} >=1.1,
-        cardano-ledger-babbage-test >=1.1,
-        cardano-ledger-binary:{cardano-ledger-binary, testlib} >=1.0,
-        cardano-ledger-conway:{cardano-ledger-conway, testlib} >=1.1,
-        cardano-ledger-conway-test >=1.1,
-        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.0,
-        cardano-ledger-pretty >=1.0,
-        cardano-ledger-mary >=1.1,
-        cardano-ledger-shelley >=1.1,
-        cardano-ledger-shelley-test >=1.1,
-        cardano-ledger-shelley-ma-test >=1.1,
-        cardano-protocol-tpraos >=1.0,
+        cardano-ledger-allegra,
+        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib},
+        cardano-ledger-alonzo-test,
+        cardano-ledger-api,
+        cardano-ledger-babbage:{cardano-ledger-babbage, testlib},
+        cardano-ledger-babbage-test,
+        cardano-ledger-binary:{cardano-ledger-binary, testlib},
+        cardano-ledger-conway:{cardano-ledger-conway, testlib},
+        cardano-ledger-conway-test,
+        cardano-ledger-core:{cardano-ledger-core, testlib},
+        cardano-ledger-pretty,
+        cardano-ledger-mary,
+        cardano-ledger-shelley,
+        cardano-ledger-shelley-test,
+        cardano-ledger-shelley-ma-test,
+        cardano-protocol-tpraos,
```

Comment:
why did you remove the bounds?

Reply:
`cardano-ledger-test` is never released, so it will always use the packages that are in the repo, which is defined by `cabal.project`

In other words, bounds are only useful for packages that are being drawn from CHaPs or hackage. So, in order to avoid redundant work it is best to remove them